templates/terraform/custom_import/compute_global_network_endpoint.go.erb

```diff
@@ -1,7 +1,7 @@
 config := meta.(*Config)
 // FQDN, port and ip_address are optional, so use * instead of + when reading the import id
 if err := parseImportId([]string{
-	"projects/(?P<project>[^/]?)/global/networkEndpointGroups/(?P<global_network_endpoint_group>[^/]+)/(?P<ip_address>[^/]+)/(?P<fqdn>[^/]*)/(?P<port>[^/]+)",
+	"projects/(?P<project>[^/]+)/global/networkEndpointGroups/(?P<global_network_endpoint_group>[^/]+)/(?P<ip_address>[^/]+)/(?P<fqdn>[^/]*)/(?P<port>[^/]+)",
```

Comment:
I think we want to replace `+` with `*` for ip_address and port as well. Thoughts?

The * allows them to be empty, as these fields are optional. The regexes below have `*` for both